assets/js/modules/analytics/components/dashboard/DashboardAllTrafficWidgetV2/DimensionTabs.js

```diff
@@ -58,7 +58,7 @@ export default function DimensionTabs( { dimensionName } ) {
 	const activeTab = tabs.findIndex( ( v ) => v.dimensionName === dimensionName );
 
 	const handleTabUpdate = useCallback( ( index ) => {
-		setValues( FORM_ALL_TRAFFIC_WIDGET, { dimensionName: tabs[ index ].dimensionName, dimensionValue: '' } );
+		setValues( FORM_ALL_TRAFFIC_WIDGET, { dimensionName: tabs[ index ].dimensionName, dimensionValue: '', dimensionColor: '' } );
```

Comment:
Let's split it into multiple lines to make it easier to read:

```suggestion
		setValues( FORM_ALL_TRAFFIC_WIDGET, {
			dimensionName: tabs[ index ].dimensionName,
			dimensionValue: '',
			dimensionColor: '',
		} );
```